coldfront/plugins/qumulo/tests/utils/mock_data.py

```diff
@@ -228,3 +230,153 @@ def set_allocation_attributes(
                 allocation=allocation,
                 value=form_data.get(allocation_attribute_name),
             )
+
+
+def get_mock_quota_base_allocations(STORAGE2_PATH: str) -> dict:
+    return {
+        f"{STORAGE2_PATH}/exlude/": {
```

Comment:
@harterjwustl 
Being specific denotes clear intent, such as not_found_in_coldfront.

Reply:
Fair.  I'll take it into consideration and probably change it back